package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "appcd-plugin-android",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
Unless we're gonna publish this soonish, maybe defer the version bump until we go to publish.